daemon/src/routes_maker.rs

```diff
@@ -304,12 +304,15 @@ mod tests {
     /// Creates an "Authorization" header that matches the password above,
     /// in particular it has been created through:
     /// ```
-    /// base64(maker:hex("Now I'm feelin' so fly like a G6"))
+    /// let password = "Now I'm feelin' so fly like a G6".to_string();
+    /// let password_vec = password.as_bytes();
+    /// let password_hex = hex::encode(vec);
+    /// let basic_auth = base64::encode(&format!("{}:{}", "itchysats", password_hex));
```

Comment:
That diff seems unnecessary if we are just changing the username from `maker` to `itchysats`.